docs/DeveloperGuide.md

```diff
@@ -241,6 +244,42 @@ _{more aspects and alternatives to be added}_
 _{Explain here how the data archiving feature will be implemented}_
 
 
+### <a name="adding_a_session"></a>Adding a session
+
+Adding a session to a class requires user input from the CLI, then the parser will parse the user input and 
+to obtain the name and the date of the session. The newly added session will also be filled with students' 
+default presence and participation.
+
+The below example given assumes that the user inputs `addses s/Tutorial 1 d/1/1/2020`, which 'Tutorial 1' and
+'1/1/2020' are the name and the date of the session to be added, and `addses` command has already been parsed.
+
+Step 1: Parse input and create session
+
+A `AddSessionCommandParser` is created by `LogicManager` and calls the parse method with the given
```

Comment:
* Should this be "An"? I've noticed this in other places as well.
* Perhaps you can use markdown `AddSessionCommand#parse()` to replace "parse method'? I've noticed this in line 272 and 277 for other methods as well.